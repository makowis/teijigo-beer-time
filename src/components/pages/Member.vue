<template>
  <tbt-default>
    <div class="member">
      <tbt-h1>MEMBER</tbt-h1>
      <table class="member-table">
        <thead>
          <tr>
            <th>名前</th>
            <th>担当</th>
            <th>自己紹介</th>
          </tr>
        </thead>
        <tbody>
          <tr v-for="member in members" v-bind:key="member.name">
            <td>{{ member.name }}</td>
            <td v-html="member.charge"></td>
            <td v-html="member.profile"></td>
          </tr>
        </tbody>
      </table>
    </div>
  </tbt-default>
</template>

<script lang="ts">
import Vue from 'vue';
import Component from 'vue-class-component';
import members from '@/resources/members';
import TbtDefault from '@/components/templates/TbtDefault.vue';
import TbtH1 from '@/components/atoms/TbtH1.vue';

@Component({
  components: {
    TbtDefault,
    TbtH1,
  },
})
export default class Member extends Vue {
  members = members;
}
</script>

<style scoped>
.member-table {
  max-width: 90%;
  margin: auto;
  border: solid 1px black;
  border-collapse: separate;
}

.member-table td,
.member-table th {
  border: solid 1px black;
  padding: 5px 10px;
}
</style>
