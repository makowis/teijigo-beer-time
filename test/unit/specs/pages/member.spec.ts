import { mount, RouterLinkStub } from '@vue/test-utils'
import Member from '@/pages/member.vue';

describe('index.vue', () => {
  const vm = mount(Member, {
    stubs: {
      NuxtLink: RouterLinkStub
    }
  }).vm;  

  it('should match the snapshot', () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
