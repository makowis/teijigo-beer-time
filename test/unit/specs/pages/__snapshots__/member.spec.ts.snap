// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue should match the snapshot 1`] = `
<div
  class="member"
>
  <h1
    class="h1"
  >
    MEMBER
  </h1>
   
  <table
    class="member-table"
  >
    <thead>
      <tr>
        <th>
          名前
        </th>
         
        <th>
          担当
        </th>
         
        <th>
          自己紹介
        </th>
      </tr>
    </thead>
     
    <tbody>
      <tr>
        <td>
          さかもとP
        </td>
         
        <td>
          なばかりプロデューサー兼テイジゴビールタイマー
        </td>
         
        <td>
          ビール大好き！！！！！！
        </td>
      </tr>
      <tr>
        <td>
          ビール飲まない社員K
        </td>
         
        <td>
          人の褌や自分の褌で相撲を取ったりする

          <br />
          
（主にトラックメーカー）
        </td>
         
        <td>
          ビールじゃなくてハイボールなら飲みます。

          <br />
          
定時後ハイボールタイム、よろしこ
        </td>
      </tr>
      <tr>
        <td>
          yokizo
        </td>
         
        <td>
          えらそーにする
        </td>
         
        <td>
          リハビリ中で戦力になれないので、えらそーにしてます。
        </td>
      </tr>
      <tr>
        <td>
          WestTail
        </td>
         
        <td>
          ProTools見習い
        </td>
         
        <td>
          周囲の素晴らしい才能をリスペクトしつつ頑張りたいと思います
        </td>
      </tr>
      <tr>
        <td>
          ケロ
        </td>
         
        <td>
          ミュージさん
        </td>
         
        <td>
          ウィットに富んだフレキシブルな音楽でナウなヤングにバカウケ
        </td>
      </tr>
      <tr>
        <td>
          Gutch
        </td>
         
        <td>
          曲に効果音を突っ込む
        </td>
         
        <td>
          ビール飲めない！！！！！
        </td>
      </tr>
      <tr>
        <td>
          みーやん
        </td>
         
        <td>
          ハモるの大好きはもりびと
        </td>
         
        <td>
          ボーカルじゃなくてコーラスが得意
        </td>
      </tr>
      <tr>
        <td>
          台湾豆乳を飲む人
        </td>
         
        <td>
          作業に参加しない係
        </td>
         
        <td>
          ﾆﾎﾝｺﾞワカリマセン
        </td>
      </tr>
      <tr>
        <td>
          マッキー
        </td>
         
        <td>
          BPM200以上
        </td>
         
        <td>
          お酒大好き！！！！！！
        </td>
      </tr>
      <tr>
        <td>
          Ag+
        </td>
         
        <td>
          テレアポ
        </td>
         
        <td>
          どこでもアポとります！
        </td>
      </tr>
      <tr>
        <td>
          管理人MAKO
        </td>
         
        <td>
          サイトの管理とお茶汲み
        </td>
         
        <td>
          サイトの管理人です。

          <br />
          
音楽作れないのでホムペ担当になりました。

          <br />
          
昔懐かしい雰囲気のサイトを作って行きたいと思います！
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
